easybuild/easyconfigs/l/LIBSVM/LIBSVM-3.24-GCCcore-10.3.0.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'MakeCp'
+
+name = 'LIBSVM'
+version = '3.24'
+
+homepage = 'https://www.csie.ntu.edu.tw/~cjlin/libsvm/oldfiles/'
+description = """LIBSVM is an integrated software for support vector classification, (C-SVC, nu-SVC), regression
+ (epsilon-SVR, nu-SVR) and distribution estimation (one-class SVM). It supports multi-class classification."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
+
+source_urls = [homepage]
```

Comment:
@stefan-wolfsheimer I would change this to

```python
source_urls = ['https://www.csie.ntu.edu.tw/~cjlin/libsvm/oldfiles/']
```

and use the original URL for `homepage`:

```
homepage = 'https://www.csie.ntu.edu.tw/~cjlin/libsvm/'
```